torch_geometric/utils/isolated.py

```diff
@@ -7,19 +7,72 @@
 from torch_geometric.utils.num_nodes import maybe_num_nodes
 
 
+def num_isolated_nodes(
+    edge_index: Tensor,
+    num_nodes: Optional[Union[int, Tuple[int, int]]] = None,
+) -> Union[int, Tuple[int, int]]:
+    r"""Returns the number of isolated nodes in the graph given by
+    :attr:`edge_index`. Please specify :obj:`num_nodes` as a tuple of two
```

Comment:
Add a comment how isolated nodes are defined, no in or no out degree?

Reply:
ok